.travis.yml

```diff
@@ -1,33 +1,61 @@
 language: php
 
-sudo: false
-
 cache:
   directories:
     - $HOME/.composer/cache/files
 
+branches:
```

Comment:
it is more fore you, to avoid having two builds triggered if you open a PR from a branch of the repo